.gitignore

```diff
@@ -14,5 +14,6 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 .env
+test-debug.env
```

Comment:
ignoring this file as I used it to run integration test locally and needed more than just .env (goland wants a visible file...)